### DISCUSSION TITLE
Welcome to LeoCAD Discussions!

### DISCUSSION THREAD

**Discussion by leozide** [leozide/leocad#541] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by nathaneltitane** [comment]
@leozide do you have a roadmap to follow for all fixes and implementation you are writing into the project?
